### ISSUE TITLE
Android > Microsoft Edge: Latest version is showing up as an unsupported browser.  Unable to reach the login page

### ISSUE THREAD

**Issue by Gustavo-Hilario** [Automattic/wp-calypso#56295]
### Quick summary

Android: Latest version of Microsoft Edge is showing up as an unsupported browser.  Users are unable to reach the login page

### Steps to reproduce

1. Go to Play Store and download the Microsoft Edge app
2. Access the app and search for "WordPress login". One of the first results should show as "Unsupported Browser - WordPress.com" you can access this link or try it directly here: https://wordpress.com/start/
3. [First, you'll see this notice](https://d.pr/i/rbYLht). Click on "Continue loading the page anyway"
4. [You shall see this blank unresponsive page now](https://d.pr/i/ZARwSc).
5. Only after refreshing the page multiple times I was able to access the login page.

### What you expected to happen

First, I would not expect that the most recent Microsoft Edge mobile browser to be an unsupported browser. If this is indeed an unsupported browser, I would expect at least to be able to reach the login page. 

### What actually happened

[The login page is showing as a blank unresponsive page](https://d.pr/i/ZARwSc).

### Context

#C03TY6J1A/p1631738946211800-sl

### Operating System

Android

### Browser

Microsoft Edge

**Comment by Robertght** [comment]
Thank you for reporting this @Gustavo-Hilario ! It appears this may be suitable for https://github.com/wordpress-mobile/WordPress-Android/issues

I've created a new issue based on this one here: https://github.com/wordpress-mobile/WordPress-Android/issues/15332

**Comment by Gustavo-Hilario** [comment]
Awesome! Thank you @Robertght 🙏

**Comment by Robertght** [comment]
I might've misunderstood this one, for now, however, I'm wondering if this will be a supported browser.

**Comment by sixhours** [comment]
Neither @mreishus or I have Android devices for testing this. @yansern is this something your team could look at?

**Comment by simison** [comment]
@Automattic/team-calypso-frameworks I think this is related to deprecating old browsers and somehow latest Edge gets unsupported as well.

**Comment by jsnajdr** [comment]
Yes, it seems that Calypso unsupported browser detection code is the culprit. To reproduce and fix this, the crucial bit of information we need is the exact `User-Agent` header that the Edge on Android browser is sending. As I don't have a testing device for that, I tried to google the Edge on Android UA string and found this page: https://www.whatismybrowser.com/guides/the-latest-user-agent/edge. According to them, the UA string is:
```
Mozilla/5.0 (Linux; Android 10; HD1913) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.80 Mobile Safari/537.36 EdgA/93.0.961.53
```
When I try this on my local Calypso, however, it parses this UA string as "Chrome Mobile 94.0.4606", the `browserslist-usergagent` library then converts this to just "Chrome 94.0.4606", and that UA matches the "last 2 Chrome versions" spec. Even if it was an older Chrome, it would still match the "Electron >= 11" spec because that includes also older Chromes back to version 87.

**Comment by noahtallen** [comment]
Interestingly, I was unable to find `EdgA` as a useragent in our Tracks data for the redirect :thinking:

**Comment by cometgrrl** [comment]
I was only able to reproduce this using chrome dev tools with the suggested user agent for Edge Android mobile:

`Mozilla/5.0 (Linux; Android 8.1.0; Pixel Build/OPM4.171019.021.D1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.109 Mobile Safari/537.36 EdgA/42.0.0.2057`

However, after clicking continue anyway, the page loaded fine. The more current UA mentioned by @jsnajdr doesn't trigger the unsupported page.

I recommend closing this unless we have other/more recent user reports. Or someone with an Android phone who can try it out directly.

**Comment by noahtallen** [comment]
> The more current UA mentioned by @jsnajdr doesn't trigger the unsupported page.

Ah, in November 2021, we changed the redirect behavior to only explicitly block browsers we know don't work. (Rather than casting a wide net with just the "most recent 2 chrome versions.") So we only forbid chrome versions under 79, because Calypso crashes for them. If the current UA is higher than that, it won't trigger the redirect anymore. (Which seems to be the case and would explain why we haven't gotten reports!)
